ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParseUriTest.java

```diff
@@ -38,27 +38,25 @@
   @Test
   public void testOutput() {
     final List<String> validInput = Arrays.asList(//
-        "{\"attributeMap\":{},\"payload\":\"\"}", //
-        "{\"attributeMap\":" //
+        "{\"attributeMap\":{},\"payload\":\"\",\"messageId\":\"0000000001\"}", "{\"attributeMap\":" //
             + "{\"uri\":\"/submit/telemetry/ce39b608-f595-4c69-b6a6-f7a436604648"
             + "/main/Firefox/61.0a1/nightly/20180328030202\"" //
-            + "},\"payload\":\"\"}",
+            + "},\"payload\":\"\",\"messageId\":\"0000000001\"}",
```

Comment:
Right now for this to the pubsub message ID is required. Otherwise if the field is missing or `null`: `org.apache.beam.sdk.Pipeline$PipelineExecutionException: java.lang.RuntimeException: org.apache.beam.sdk.coders.CoderException: cannot encode a null String`

Reply:
I don't think there's going to be a way around this (it's true that any value that Beam is going to encode/decode cannot be null), which is the potentially really tricky part of this change. It may require updating all of our test cases to use messageIds, which would be onerous.